packages/fast-app-shell-react/src/app-link/index.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import manageJss, {
+    ComponentStyles,
+    ManagedClasses,
+} from "@microsoft/fast-jss-manager-react";
+import {
+    accentFillRest,
+    cornerRadius,
+    DesignSystem,
+    DesignSystemResolver,
+    designUnit,
+    neutralFillStealthActive,
+    neutralFillStealthHover,
+    neutralFillStealthRest,
+    neutralForegroundRest,
+} from "@microsoft/fast-components-styles-msft";
+import { NavLink } from "react-router-dom";
+import { get } from "lodash-es";
+import { multiply, toPx } from "@microsoft/fast-jss-utilities";
+
+export interface DasbhoardLinkClassNameContract {
+    dashboardLink: string;
+    dasboardLink__active: string;
+}
+
+export interface AppLinkProps {
+    href: string;
+    title: string;
+    children: React.ReactNode;
+}
+
+const styles: ComponentStyles<DasbhoardLinkClassNameContract, DesignSystem> = {
+    dashboardLink: {
+        position: "relative",
+        display: "flex",
+        width: toPx(multiply(designUnit, 10)),
+        height: toPx(multiply(designUnit, 10)),
+        alignItems: "center",
+        justifyContent: "center",
+        background: neutralFillStealthRest,
+        transition: "all .05s ease-out",
+        "& svg": {
+            stroke: neutralForegroundRest,
+        },
+        "&:hover": {
+            background: neutralFillStealthHover,
+        },
+        "&:active": {
+            background: neutralFillStealthActive,
+        },
+    },
+    dasboardLink__active: {
+        "&::before": {
+            position: "absolute",
+            top: "8px",
+            left: "4px",
```

Comment:
ltr?

Reply:
This is actually centered so I'm not going to worry too much about it.